packages/devextreme/testing/tests/DevExpress.ui.widgets.form/form.tests.js

```diff
@@ -4364,3 +4367,114 @@ QUnit.test('TagBox.SelectionChanged is raised once if formData is wrapped into a
     assert.deepEqual(formData, { arrayField: ['item1'] }, 'formData');
     assert.strictEqual(onSelectionChangedCounter, 1, 'onSelectionChangedCounter');
 });
+
+[['dxCalendar', new Date(2019, 1, 2), { dxCalendar: new Date(2019, 1, 3) } ],
+    ['dxRangeSlider', [1, 5], { dxRangeSlider: [1, 3] } ],
+    ['dxSlider', 199, { dxSlider: 99 }],
+    ['dxSwitch', true, { dxSwitch: false }],
+    ['dxAutocomplete', '1', { dxAutocomplete: '2' }],
+    ['dxColorBox', new Date(2019, 1, 1), { dxColorBox: new Date(2019, 1, 2) } ],
+    ['dxDateBox', new Date(2017, 0, 3), { dxDateBox: new Date(2019, 1, 2) }],
+    ['dxDropDownBox', '1', { dxDropDownBox: '3' }],
+    ['dxHtmlEditor', '<p>a</p>', { dxHtmlEditor: '<p>b</p>' }],
+    ['dxLookup', '1', { dxLookup: '3' }],
+    ['dxNumberBox', '1', { dxNumberBox: '3' }],
+    ['dxRadioGroup', '1', { dxRadioGroup: '3' }],
+    ['dxSelectBox', '1', { dxSelectBox: '2' }],
+    ['dxTagBox', ['1'], { dxTagBox: ['2'] }],
+    ['dxTextArea', 'a', { dxTextArea: 'b' }],
+    ['dxTextBox', 'a', { dxTextBox: 'b' }],
+    ['dxDateRangeBox', [new Date(2021, 8, 17), new Date(2021, 9, 17)], { dxDateRangeBox: [null, null] } ],
+    // TODO: implement for renovated component check_box.tsx
```

Comment:
Why we should leave this comment?

Reply:
we decide to implement it in separate PR